app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,22 @@
+class RentsController < ApplicationController
+  respond_to :json
+  before_action :authenticate_user!
+
+  def index
+    rent = current_user.rents
+    if rent == []
```

Comment:
This comparison can be made in a cleaner way using the `.empty?` method. Generally it is better to use a built in method rather than a comparison using `==`, should there be one.

Reply:
Thanks! Done